lib/git-process.ts

```diff
@@ -228,7 +228,7 @@ export class GitProcess {
 
       if (options && options.stdin !== undefined) {
         // See https://github.com/nodejs/node/blob/7b5ffa46fe4d2868c1662694da06eb55ec744bde/test/parallel/test-stdin-pipe-large.js
-        spawnedProcess.stdin.end(options.stdin, options.stdinEncoding)
+        spawnedProcess.stdin!.end(options.stdin, options.stdinEncoding)
```

Comment:
This change is a result of replacing the old lockfile which upgrades the project to the latest `@types/node`. #309 is the minimal repro of this particular issue. Could can pin `@types/node` to `11.9.6` so it will simplify this diff?

Reply:
Pinning node to `11.9.6` fixes that problem.